src/lib.rs

```diff
@@ -4,14 +4,14 @@ extern crate objc;
 use objc::runtime::Object;
 
 #[cfg(target_os = "macos")]
-pub mod macos;
+pub mod appkit;
 #[cfg(target_os = "ios")]
-pub mod ios;
+pub mod uikit;
```

Comment:
@msiglreith I've been looking into reviving the related `ash-window` PR now that `raw-window-handle 0.5` is out _and_ used in `winit 0.27`, and stumbled upon these cfgs. According to https://github.com/rust-windowing/raw-window-handle/pull/70/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759 (surprisingly the author detailed/justified nothing of this in PR/commit messages...) it should be possible to use these across `macos` _and_ `ios` now when targeting "Mac Catalyst"?